lib/elixir/test/elixir/process_test.exs

```diff
@@ -170,6 +170,17 @@ defmodule ProcessTest do
     end
   end
 
+  describe "set_label/1" do
+    test "sets a process label, compatible with OTP 27+ `:proc_lib.get_label/1`" do
+      label = {:some_label, :random.uniform(99999)}
+      assert :ok = Process.set_label(label)
+
+      if :erlang.system_info(:otp_release) == ~c"27" do
```

Comment:
I think we should use `>=` (also using the Elixir version for [consistency](https://github.com/elixir-lang/elixir/pull/13397))
```suggestion
      if System.otp_release() >= "27" do
```

Reply:
I like using `System.otp_release/0`. Using `>=` without first converting the values to integers feels wrong to me, although it doesn't fail for larger release values until we get to `"100" >= "27"`. I'm sure this code won't be here that long. 😅